ddtrace/contrib/asyncio/__init__.py

```diff
@@ -1,7 +1,9 @@
 """
-This integration provides the ``AsyncioContextProvider`` that follows the execution
-flow of a ``Task``, making possible to trace asynchronous code built on top
-of ``asyncio``. To trace asynchronous execution, you must::
+This integration provides a legacy context provider for supporting tracing of
```

Comment:
I would reverse the ordering so that the automatic instrumentation is mentioned before the legacy